### ISSUE TITLE
Fine-grained muting options

### ISSUE THREAD

**Issue by Paarsec** [bluesky-social/social-app#6330]
### Describe the Feature

I believe that users often unfollow others because they are:

1) posting too much about a current event,
2) posting too much about a specific topic,
3) reposting too much,
4) posting too often in the same long thread.

When I mute somebody I frequently forget they exist indefinitely, or can't find them anymore in the long list of permanent mutes I have. 

I suggest we implement a few new muting options to lessen the amount of unfollowing due to these issues, to preserve bluesky relationships long term. 

Here are potential solutions for each of the above issues:

1) Ability to mute a user for a set amount of time. Mastodon has this option. I'd suggest a submenu under the mute option, called "mute for...", with a list of options like 3hrs, 24hrs, 1 week;
2) being able to mute the posts of one user that contain a specific hashtag. This would incentivize the usage of hashtags so people can mute specific topics if desired;
3) A "Mute reposts by this user" option;
4) An option to mute a thread from feed views (the thread would remain visible in profile views so that it can be unmuted if desired).

Thank you!

**Comment by serenebabe** [comment]
Mute reposts would be so great!

**Comment by surfdude29** [comment]
Muting reposts is definitely planned and @/gaeron [gave an update](https://github.com/bluesky-social/social-app/issues/1116#issuecomment-2344808590) in September saying:

> We're definitely aware of the importance of this feature — blocked on a few things on the backend so we can't prioritize it right now, but we're planning to get to it as soon as those blockers are solved.

**Comment by whetstone303** [comment]
This issue touches on things I have been wishing for (muting reposts from a specific user, time-limited muting) as well as things I hadn't thought of (muting posts containing a hashtag or keyword for a specific user). Very much hope it's prioritized!

**Comment by nielsle** [comment]
Perhaps introduce a rate limit for frequent posters, so that I only see one or two of their posts every week.
But this should be hidden behind a user friendly description such as: "See less posts like this"

**Comment by cooljeanius** [comment]
> Perhaps introduce a rate limit for frequent posters, so that I only see one or two of their posts every week. But this should be hidden behind a user friendly description such as: "See less posts like this"

Kinda related: #6432, although that's for interactions rather than feed-browsing...

**Comment by SystemsOperator** [comment]
Agreed, necessary!

Just adding a ticket that looks like it's asking for the same thing: (there are likely more)
https://github.com/bluesky-social/social-app/issues/6413

**Comment by cooljeanius** [comment]
> And, unsure if this should be a separate ticket or could be considered a fine-grained option:
> 
>     * **Mute YOURSELF (your own posts, others' reposts of you) for another user.**
>       Unlike blocking, this shouldn't be public. Rather, a safer way to lessen chances of interaction with some users.

I had this idea, too, but on further consideration realized it could potentially be used as a vector for harassment: someone could use the option to hide a harassment campaign they're directing from their target, resulting in the target not being able to figure out where everybody's coming from. If this concern can be overcome, though, I'd agree it'd be a valuable option.

**Comment by SystemsOperator** [comment]
> > And, unsure if this should be a separate ticket or could be considered a fine-grained option:
> > 
> > * **Mute YOURSELF (your own posts, others' reposts of you) for another user.**
> >   Unlike blocking, this shouldn't be public. Rather, a safer way to lessen chances of interaction with some users.
> 
> I had this idea, too, but on further consideration realized it could potentially be used as a vector for harassment: someone could use the option to hide a harassment campaign they're directing from their target, resulting in the target not being able to figure out where everybody's coming from. If this concern can be overcome, though, I'd agree it'd be a valuable option.

That's a good point; I wouldn't even have thought of using it like that.
That's really unfortunate.

**Comment by dartiss** [comment]
Would love to particularly see a mute reports features for Bluesky - I often mute or remove people who excessively report, whereas would, genuinely, like to see their own posts.
